src/main/java/apoc/load/LDAPResult.java

```diff
@@ -0,0 +1,11 @@
+package apoc.load;
+
+import java.util.Map;
+
+public class LDAPResult {
```

Comment:
I only did this to have the name 'entry' which is more reflecting the ldap name LDAPEntry. But technically we could also use MapResult. Then the examples have to be changed.